tests/Rector/ClassMethod/AddRouteAnnotationRector/config/configured_rule.php

```diff
@@ -3,10 +3,19 @@
 declare(strict_types=1);
 
 use Rector\Config\RectorConfig;
+use Rector\Symfony\Contract\Bridge\Symfony\Routing\SymfonyRoutesProviderInterface;
 use Rector\Symfony\Rector\ClassMethod\AddRouteAnnotationRector;
+use Rector\Symfony\Tests\Rector\ClassMethod\AddRouteAnnotationRector\Source\DummySymfonyRoutesProvider;
 
 return static function (RectorConfig $rectorConfig): void {
     $rectorConfig->import(__DIR__ . '/../../../../../config/config.php');
+
     $rectorConfig->rule(AddRouteAnnotationRector::class);
-    $rectorConfig->symfonyRoutesJson(__DIR__ . '/route.json');
+
+    // only for tests
+    $services = $rectorConfig->services();
+    $services->set(DummySymfonyRoutesProvider::class);
+
+    // give this service autowiriting preferences
```

Comment:
should `autowiriting` be `autowiring` ?

```suggestion
    // give this service autowiring preferences
```